doc/HACKING.md

```diff
@@ -290,6 +290,35 @@ It can be used to analyze the lightningd source code by running
 binaries) is stored in .nccout files. You can browse it, for instance, with
 a command like `nccnav lightningd/lightningd.nccout`.
 
+Code Coverage
+-------------
+Code coverage can be measured using Clang's source-based instrumentation.
+
+First, build with the instrumentation enabled:
+```shell
+make clean
+./configure --enable-clang-coverage CC=clang
+make -j
```

Comment:
```suggestion
make -j
```
Just a followup question.

It has some special meaning, or it is equal to `make`? Usually here is expected that it is missing the number of processors. like `make -j$(nproc)`.

Reply:
`make -j` without nproc allows "unlimited" parallel jobs.  This is how I usually build CLN, but maybe it would cause problems for machines with less CPU and RAM.

Added `nproc` to be safe.

Reply:
Oh I did not know this `unlimited` stuff, thanks